doc/source/whatsnew/v0.18.1.txt

```diff
@@ -131,6 +131,7 @@ API changes
 
 
 
+- the default for ``.query()/.eval()`` is now ``engine=None`` which will use ``numexpr`` if its installed, else will fallback to the ``python`` engine. This mimics the pre-0.18.1 behavior if ``numexpr`` is installed (and which previously would raise if ``numexpr`` was NOT installed and ``.query()/.eval()`` was used). (:issue:`12749`)
```

Comment:
The issue of showing a ``PerformanceWarning`` (good idea) is that we have to be careful when doing it, because for example string ops are by definition in python, so will create a separate issue for that.